cloud/installation/poller-troubleshoot.md

```diff
@@ -0,0 +1,264 @@
+
+```shell
+/var/log/centreon-gorgone/gorgoned.log
+```
+
+### Step 4: Set Gorgone log level to info or debug
+
+The default log level for Gorgone is **error**. If you need more details about what Gorgone is doing:
+
+1. Edit the following file:
+
+   ```shell
+   /etc/sysconfig/gorgoned
+   ```
+
+2. Change the **--severity** parameter value to **info** or **debug**.
+
+3. Restart the **gorgoned** process:
+
+   ```shell
+   systemctl restart gorgoned
+   ```
+
+### Step 5: Check Gorgone's connection
+
+Once you have set the log level to info or debug, this INFO message should appear in Gorgone logs:
+
+```shell
+INFO - [pullwss] websocket connected
+```
+
+If you get another message, this means that there is a problem with the network configuration or an authentication problem.
+
+### Step 6: Check that the deploy command works
+
+Check the log messages generated by Gorgone while the configuration is being deployed. The log should contain the following lines, which mean that the poller has received the configuration.
+
+```shell
+2024-09-13 14:28:50 - INFO - [action] Copy processing - Received chunk for '/etc/centreon-engine//'
+2024-09-13 14:28:51 - INFO - [action] Copy processing - Copy to '/etc/centreon-engine//' finished successfully
+2024-09-13 14:28:51 - INFO - [action] Copy processing - Received chunk for '/etc/centreon-broker/'
+2024-09-13 14:28:51 - INFO - [action] Copy processing - Copy to '/etc/centreon-broker/' finished successfully
+```
+
+If all previous steps were OK but the log does not show that the poller has received the configuration, contact [our support team](https://support.centreon.com/hc/en-us).
+
+### Step 7: Check that configuration files are correctly updated
+
+Check the modification dates of configuration files: if you see that the files were not updated when you deployed the configuration, check that the **gorgone** user has rights on this folder.
+
+```shell
+ls -l /etc/centreon-engine
+```
+
+### Step 8: Check that Engine reloads or restarts correctly
+
+Check Engine's log file according to the method you have selected when deploying the configuration.
+
+1. Open the log file:
+
+```shell
+vi /var/log/centreon-engine/centengine.log
+```
+
+2. [Deploy the configuration](../monitoring/monitoring-servers/deploying-a-configuration.md) from the interface using either the **reload** or **restart** method.
+
+3. Check the log file for the following messages:
+
+* If you have selected the **reload** method, the log file should contain:
+
+```shell
+[process] [info] [646] Starting to reload configuration.
+(...)
+[process] [info] [646] Reload configuration finished.
+```
+
+* If you have selected the **restart** method, the log file should contain:
+
+```shell
+[process] [info] [3142] Centreon Engine yy.mm.p starting ...
+```
+
+If it doesn't, check that **gorgoned** has the right to write in `/etc/centreon-engine` and that **centengine** has the right to read in it.
```

Comment:
This sentence was what step 7 was supposed to check; do we still need it there?

Reply:
Correct.
In this case, there is no clear action on next steps, it depends on other logs messages.
```suggestion
If it doesn't, check errors in logs to get more information about any issue.
```